### ISSUE TITLE
Add info from UCD survey into Hubspot

### ISSUE THREAD

**Issue by o-ram** [cal-itp/customer-success#487]
When Cal-ITP and UC Davis did a survey together, they asked agencies what initiatives they were interested in. The CS team received a spreadsheet of agencies who provided answers to these questions.

We should add this info to Hubspot in some way. What/how is TBD. Until then, the spreadsheet is [here](https://docs.google.com/spreadsheets/d/1JZJMTU0HLeiiLy-zoI2qOpGLee5cFszu/edit?usp=drive_web&ouid=107888682452903153368&rtpof=true) for reference.

**Comment by o-ram** [comment]
cc @mariearaneta (this is the spreadsheet I just mentioned in a comment to you), @ninadinh-la, @laneymangan

**Comment by o-ram** [comment]
@ninadinh-la @mariearaneta @laneymangan I want to see if we can close this issue, so **could everyone please [check the box off ](https://docs.google.com/spreadsheets/d/1JZJMTU0HLeiiLy-zoI2qOpGLee5cFszu/edit#gid=1967672024)for each agency that has this info in their Hubspot profile?** And add the info in where it hasn't been done yet? Once all the rows are checked off, I think we can close this out.

Also, since we were just discussing boosting emails, referencing interest expressed in this survey is a great way to do so if you didn't already do so in the initial email. You could say something like "You either completed this survey, or your name was provided as a point of contact" to refresh people's memories.

Thanks, all!

**Comment by ninadinh-la** [comment]
@o-ram I checked off and completed all of my agencies and highlighted them in purple as well. If I missed an agency that's assigned to me, let me know!

**Comment by mariearaneta** [comment]
@o-ram My agencies are highlighted in yellow and I've added all info into Hubspot. Please let me know if I missed anything or need to make any adjustments.

**Comment by o-ram** [comment]
Thanks, all, looks good! I added in the information for all the agencies without an assigned person and did some cleanup as I went (mostly merging of some duplicate deals)

Notes have been added containing the information provided by each agency, and in the event they expressed interest in an initiative where a pipeline exists (such as payments or Benefits), a deal was also created. Therefore, I am closing this issue with this comment.
